index.js

```diff
@@ -1,52 +1,111 @@
-//Now in this Exercise we want you to create 3 buttons wich clicks actions are going to hide the respective question above with their aswer and if they are hidden then the click option should be display the question with their respective aswer
-// append the 3 buttons to the element with the 'id = a-4' as childrens
-
-
-//resolve // QUESTION 4 here
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
+const  calculate_sphere = () =>
+ {
+  // declare volume and surface vaiables
+  var volume;
+  var surface;
+  //get radius element and value from html document and assign it to radius variable, get the absolute value for the value.
+  var radius = document.getElementById('radius').value;
+  radius = Math.abs(radius);
+
+  //calculate the volume, fix digits after the dot to 4, then make that value equal to "volume" element value in the html document
+  volume = (4/3) * Math.PI * Math.pow(radius, 3);
+  volume = volume.toFixed(4);
+  document.getElementById('volume').value = volume;
+
+  //calculate the surface, fix digits after the dot to 4, then make that value equal to "area" element value in the html document
+  surface = (4) * Math.PI * Math.pow(radius, 2);
+  surface = surface.toFixed(4);
+  document.getElementById('area').value = surface;
+  
+  return false;
+ } 
+window.onload = document.getElementById('MyForm').onsubmit = calculate_sphere;
+
+
+
+/*-----------------------------------Question 4 ------------------------------------------------------------------------------------------------------------------------------------------------
+Now in this Exercise we want you to create 3 buttons wich clicks actions are going to hide the  respective question above with their aswer and if they are hidden then the click option should 
+//be display the question with their respective aswer append the 3 buttons to the element with the 'id = a-4' as childrens
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
+    // /*for each question I did: pick question element from html document, 
+    //                            pick the button element
+    //                            call event listener: click function for the button 
+    //                            then toggle question display, between block and display.
+
+    let question1 = document.getElementById('question1');
+    let showHide1= document.getElementById('showHide1');
+    showHide1.addEventListener('click', ()  => 
+    {
+        question1.classList.toggle('visible');
+    } );
+
+
+    let question2 = document.getElementById('question2');
+    let showHide2= document.getElementById('showHide2');
+    showHide2.addEventListener('click', ()  => 
+    {
+        question2.classList.toggle('visible');
```

Comment:
Does this add and subtract the class visible which you wrote CSS for? That's a good way to do it actually